apps/files/src/components/FileFilterMenu.vue

```diff
@@ -1,26 +1,27 @@
 <template>
-  <v-menu class="mt-2" offset-y>
-    <v-btn slot="activator" flat @click="focusFilenameFilter"><v-icon large>filter_list</v-icon></v-btn>
-    <v-list>
-      <v-list-tile v-for="(filter, fid) in fileFilter" :key="fid">
-        <v-list-tile-title v-text="fileFilterTranslations[filter.name]"></v-list-tile-title>
-        <v-checkbox :input-value="filter.value" @change="setFileFilter({ name: filter.name, value: $event })"></v-checkbox>
-      </v-list-tile>
-      <v-list-tile>
-        <v-list-tile-title>
-          <span v-translate>Name Filter</span>
-        </v-list-tile-title>
-        <v-list-tile-title>
-        <search-bar @input="setFilterTerm" :value="filterTerm" ref="filenameFilter" autofocus :label="$gettext('Filter')" icon="false"/>
-        </v-list-tile-title>
-      </v-list-tile>
-    </v-list>
-  </v-menu>
+  <oc-drop toggle="#oc-filter-list-btn" mode="click">
+    <ul class="uk-list">
+      <oc-filter-menu-item v-for="(filter, fid) in fileFilter" :key="fid" :id="'oc-filter-' + filter.tag" :name="filter.tag" :value="filter.value" @change="setFileFilter({ name: filter.name, value: !filter.value })">
```

Comment:
should be an oc-nav-item

this is what you have agreed on with @felixheidecke last week - right?

Reply:
I would prefer not to use oc-nav-item since it is `<a>` tag. But anyway, you're right with the usage of oc-filter-menu-item - this is not exactly component that could be used in any other case then filtering. Gonna chat about it with @felixheidecke if it makes sense to keep component that would basically be only checkbox and label.

Reply:
I got rid of the oc-filter-menu-item and replaced it with normal `<li>`